plugin/src/main/java/com/iridium/iridiumcore/Item.java

```diff
@@ -20,6 +21,7 @@
     public String headOwner;
     public List<String> lore;
     public Integer slot;
+    public UUID headUuidOwner;
```

Comment:
To keep naming consistent, this should probably be named like this
```suggestion
    public UUID headOwnerUUID;
```